src/protocolsupport/zplatform/pe/PECreativeInventory.java

```diff
@@ -0,0 +1,135 @@
+		skipMaterials.add(Material.REDSTONE_WIRE.getId());
+		skipMaterials.add(Material.GLOWING_REDSTONE_ORE.getId());
+		skipMaterials.add(Material.BARRIER.getId());
+		skipMaterials.add(Material.BURNING_FURNACE.getId());
+		skipMaterials.add(Material.SOIL.getId());
+		skipMaterials.add(Material.POTATO.getId());
+		skipMaterials.add(Material.CARROT.getId());
+		skipMaterials.add(Material.MAP.getId());
+		skipMaterials.add(Material.DOUBLE_STEP.getId());
+		skipMaterials.add(Material.DOUBLE_STONE_SLAB2.getId());
+		skipMaterials.add(Material.PURPUR_DOUBLE_SLAB.getId());
+		skipMaterials.add(Material.WOOD_DOUBLE_STEP.getId());
+
+		// Items that crashes the client (for some reason that we don't know yet... maybe we should blame mojang for this)
+		skipMaterials.add(Material.DROPPER.getId());
+		skipMaterials.add(Material.BEETROOT.getId());
+		skipMaterials.add(Material.BEETROOT_SEEDS.getId());
+		skipMaterials.add(Material.BEETROOT_SOUP.getId());
+		skipMaterials.add(Material.GOLD_RECORD.getId());
+		skipMaterials.add(Material.GREEN_RECORD.getId());
+		skipMaterials.add(Material.RECORD_3.getId());
+		skipMaterials.add(Material.RECORD_4.getId());
+		skipMaterials.add(Material.RECORD_5.getId());
+		skipMaterials.add(Material.RECORD_6.getId());
+		skipMaterials.add(Material.RECORD_7.getId());
+		skipMaterials.add(Material.RECORD_8.getId());
+		skipMaterials.add(Material.RECORD_9.getId());
+		skipMaterials.add(Material.RECORD_10.getId());
+		skipMaterials.add(Material.RECORD_11.getId());
+		skipMaterials.add(Material.RECORD_12.getId());
+
+		Map<Material, Integer> subIds = new HashMap<Material, Integer>();
+
+		subIds.put(Material.STONE, 6);
+		subIds.put(Material.WOOD, 5);
+		subIds.put(Material.SAPLING, 5);
+		subIds.put(Material.SAND, 1);
+		subIds.put(Material.LOG, 3);
+		subIds.put(Material.LEAVES, 3);
+		subIds.put(Material.SPONGE, 1);
+		subIds.put(Material.SANDSTONE, 2);
+		subIds.put(Material.WOOL, 15);
+		subIds.put(Material.STAINED_GLASS, 15);
+		subIds.put(Material.STAINED_GLASS_PANE, 15);
+		subIds.put(Material.STAINED_CLAY, 15);
+		subIds.put(Material.CONCRETE, 15);
+		subIds.put(Material.CONCRETE_POWDER, 15);
+		subIds.put(Material.CARPET, 15);
+		subIds.put(Material.RED_ROSE, 8);
+		subIds.put(Material.STEP, 7);
+		subIds.put(Material.SMOOTH_BRICK, 3);
+		subIds.put(Material.WOOD_STEP, 5);
+		subIds.put(Material.COBBLE_WALL, 1);
+		subIds.put(Material.QUARTZ_BLOCK, 2);
+		subIds.put(Material.LEAVES_2, 1);
+		subIds.put(Material.LOG_2, 1);
+		subIds.put(Material.PRISMARINE, 2);
+		subIds.put(Material.DOUBLE_PLANT, 5);
+		subIds.put(Material.RED_SANDSTONE, 2);
+
+		List<ItemStackWrapper> itemStacks = new ArrayList<ItemStackWrapper>();
+
+		for (Material material : Material.values()) {
+			if (!skipMaterials.contains(material.getId())) {
+				ItemStackWrapper wrapper = ServerPlatform.get().getWrapperFactory().createItemStack(material);
+				if (wrapper.getType() == Material.AIR) {
+					continue;
```

Comment:
Is this neccesary? Air is already in the skiptable?

Reply:
@7kasper derp, I don't even know why that check is there